applications/serial_lte_modem/src/gpio/slm_at_gpio.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef SLM_AT_GPIO_
+#define SLM_AT_GPIO_
+
+/**@file slm_at_GPIO.h
+ *
+ * @brief Vendor-specific AT command for GPIO service.
+ * @{
+ */
+
+#include <drivers/gpio.h>
+
+#define MAX_GPIO_PIN 31
+
+/* Regular GPIO */
+#define SLM_GPIOC_OP_DISABLE	0	/* Disables pin for both input and output. */
+#define SLM_GPIOC_OP_OUT		1	/* Enables pin as output. */
+#define SLM_GPIOC_OP_IN_PU	21	/* Enables pin as input. Use internal pull up resistor. */
+#define SLM_GPIOC_OP_IN_PD	22	/* Enables pin as input. Use internal pull down resistor. */
+
+/**@brief GPIO operations. */
+enum slm_gpio_operations {
+	SLM_GPIO_OP_WRITE,
+	SLM_GPIO_OP_READ,
+	SLM_GPIO_OP_TOGGLE
+};
+
+/**
+ * @brief Initialize GPIO AT command parser.
+ *
+ * @retval 0 If the operation was successful.
+ *           Otherwise, a (negative) error code is returned.
+ */
+int slm_at_gpio_init(void);
+
+/**
+ * @brief Uninitialize GPIO AT command parser.
+ *
+ * @retval 0 If the operation was successful.
+ *           Otherwise, a (negative) error code is returned.
+ */
+int slm_at_gpio_uninit(void);
+
+/**
+ * @brief GPIO configuration.
+ *
+ * @retval 0 If the operation was successful.
+ *           Otherwise, a (negative) error code is returned.
+ */
+int do_gpio_pin_configure_set(uint16_t op, gpio_pin_t pin);
```

Comment:
this is an internal function and not an API

Reply:
Agree, remove declaration.